src/services/validatorSchema.ts

```diff
@@ -0,0 +1,33 @@
+import * as yup from "yup";
+
+export const UserSchema = yup.object({
+    user_id: yup
+        .string()
+        .strict()
+        .typeError("user_id must be a string.")
+        .defined("The user_id (string) must be provided.")
+});
+
+export const ItemSchema = yup.object({
```

Comment:
this is not validating an Item, rather a search by ID request, let's rename this to `SearchByUserIdRequest`?